pkg/api/graph.go

```diff
@@ -560,3 +560,25 @@ func LinkForImage(imageStream, tag string) StepLink {
 		return nil
 	}
 }
+
+// ClusterClaimLink determines what dependent link is required
+// for a job to claim a cluster
+func ClusterClaimLink() StepLink {
+	return &clusterClaimLink{}
+}
+
+type clusterClaimLink struct {
+}
+
+func (_ *clusterClaimLink) UnsatisfiableError() string {
+	return ""
+}
+
+func (_ *clusterClaimLink) SatisfiedBy(other StepLink) bool {
```

Comment:
It is satisfied by itself?

Reply:
Yes. Similar to this

https://github.com/openshift/ci-tools/blob/1fbc5d260e3afe92e86591f43594bb0b7459a6a5/pkg/api/graph.go#L82-L89

First time for me to touch the graph. My understanding is:
- A step (in this case, multi-stage step) requires such a link: it means the step wont run until some other step creates it.
https://github.com/openshift/ci-tools/pull/1837/files#diff-da42b74c690675134a34fba17070b2705bdf546b16b6e84e06ef9440f637a166R217
- a cluster claim step provides it by 
https://github.com/openshift/ci-tools/pull/1837/files#diff-67951af79cb529c491eaa6d7fca53e175eb141fbc9df0af3cd74c86eacf9377bR53